Detectors/ITSMFT/ITS/workflow/src/ThresholdCalibratorSpec.cxx

```diff
@@ -288,7 +299,7 @@ bool ITSCalibrator::findThresholdFit(
 // data is the number of trigger counts per charge injected;
 // x is the array of charge injected values;
 // NPoints is the length of both arrays.
-bool ITSCalibrator::findThresholdDerivative(
+bool ITSThresholdCalibrator::findThresholdDerivative(
   const char* data, const short int* x, const short int& NPoints,
```

Comment:
You can save some cpu time by using a stack array instead of `new[]/delete[]` at https://github.com/AliceO2Group/AliceO2/blob/c8432bd40a03211b20849c9c6bba61ff8ecc8045/Detectors/ITSMFT/ITS/workflow/src/ThresholdCalibratorSpec.cxx#L318, i.e. simply `float derivfloat[deriv_size];`

Reply:
good catch, done.